sympy/combinatorics/pc_groups.py

```diff
@@ -223,8 +223,6 @@ def subword_index(self, word, w):
                 low = i
                 high = i+len(w)
                 break
-        if low == high == -1:
```

Comment:
I think that the author may have left such code because -1 is likely be error output. However, I generally agree that the code shouldn't be written have such redundancy but documentation could be updated

Reply:
Is it acceptable to add to the docstring in this manner?

```diff
diff --git a/sympy/combinatorics/pc_groups.py b/sympy/combinatorics/pc_groups.py
index 65c785ae33..abf7b82258 100644
--- a/sympy/combinatorics/pc_groups.py
+++ b/sympy/combinatorics/pc_groups.py
@@ -196,7 +196,7 @@ def subword_index(self, word, w):

         (i, j)
             A tuple containing starting and ending index of ``w``
-            in the given word.
+            in the given word. If not exists, (-1,-1) is returned.

         Examples
         ========
@@ -214,6 +214,9 @@ def subword_index(self, word, w):
         >>> w = x1**7
         >>> collector.subword_index(word, w)
         (2, 9)
+        >>> w = x1**8
+        >>> collector.subword_index(word, w)
+        (-1, -1)

         """
         low = -1
```

Reply:
Yes, I think that the example can be useful.